### DISCUSSION TITLE
How do i further improve the existing model resent_arc50 by retraining?

### DISCUSSION THREAD

**Discussion by OnlinePage** [Ant-Brain/EfficientWord-Net#55] [Q&A]
Yoo! @TheSeriousProgrammer , I would like to know how can I further improve the existing model resent_arc50 by training it on custom dataset, any references or example would be a great help🙏

**Comment by OnlinePage** [comment]
@TheSeriousProgrammer day 1 of requesting a reply!!

**Comment by TheSeriousProgrammer** [comment]
Hey @OnlinePage here is the code I have for the trainer https://github.com/Ant-Brain/EfficientWord-Net-Trainer

Unfortunately the dataset we created for the training got deleted with academic vm we where using, in this you need to start from simple_trainer.py

**Reply by OnlinePage** [reply]
Thanks for reply!!
I will look forward thanks for sharing the repo.
Also have compiled efficient-word-net for kotlin and node js and they works realllly awesome out of the box with onnx runtime in both!!
Just wantttt to work on model improvisation to make more accurate and stable.
in nutshell f***k picov**ce n others.
Thanks again bud for this repo🫡.

**Reply by TheSeriousProgrammer** [reply]
That's awesome bro!! Keep us posted!
